internal/amplitude/types.go

```diff
@@ -0,0 +1,56 @@
+package amplitude
+
+import (
+	"encoding/json"
+
+	"github.com/sourcegraph/sourcegraph/internal/featureflag"
+)
+
+// AmplitudeEventPayload represents the structure of the payloads we send to Amplitude
+// when we log an event. We can send more than one event in Events, but we currently
+// send events in individual payloads.
+type AmplitudeEventPayload struct {
```

Comment:
```suggestion
type EventPayload struct {
```

Same for other occurrences, because we're already within the `amplitude` package.